examples/gnrc_border_router/main.c

```diff
@@ -18,21 +18,85 @@
  * @}
  */
 
+#include <assert.h>
 #include <stdio.h>
 
+#include "event.h"
+#include "net/dhcpv6/client.h"
+#include "net/ipv6/addr.h"
+#include "net/gnrc.h"
+#include "net/gnrc/ipv6/nib/ft.h"
 #include "shell.h"
 #include "msg.h"
 
 #define MAIN_QUEUE_SIZE     (8)
+static char _dhcpv6_client_stack[DHCPV6_CLIENT_STACK_SIZE];
 static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
 
+void *_dhcpv6_client_thread(void *args)
+{
+    ipv6_addr_t addr = {
+            .u8 = { 0xfe, 0x80, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1 }
+        };
+
+    event_queue_t event_queue;
+    gnrc_netif_t *border_interface = NULL;
+    gnrc_netif_t *wpan_interface = NULL;
+    gnrc_netif_t *netif = NULL;
+
+    (void)args;
+    /* iterate over all interfaces */
+    while ((netif = gnrc_netif_iter(netif))) {
+        /* check if interface is wired */
+        int is_wired = gnrc_netapi_get(netif->pid, NETOPT_IS_WIRED, 0, NULL, 0);
```

Comment:
Instead of looking for a wired interface, let's rather look for a non-6lo interface, so we can e.g. use an esp32 as a WiFi <-> WPAN gateway.

Reply:
I _guess_ in the scope of the `NETOPT_IS_WIRED` option, WiFi can be understood as wired?

Reply:
I mean it looks a lot like *Ether*net, but usually I would not consider WiFi to be wired.